tests/unit/controlgroup/controlgroup_options.js

```diff
@@ -0,0 +1,113 @@
+module( "Controlgroup: options" );
+
+function hasCornerClass( element ) {
+	return !!element.attr( "class" ).match( /ui-corner/g );
+}
+
+test( "disabled", function() {
+	expect( 4 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "option", "disabled", true );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after setting disabled true" );
+	strictEqual( element.find( ".ui-state-disabled" ).length, 6,
+		"Child widgets are disabled" );
+
+	element.controlgroup( "option", "disabled", false );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after setting disabled false" );
+	strictEqual( element.find( ".ui-state-disabled" ).length, 0,
+		"Child widgets are not disabled" );
+
+});
+test( "items - null", function() {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup({
+		items: {
+			"button": null,
+			"selectmenu": null,
+			"checkboxradio": null
+		}
+	});
+
+	strictEqual( element.children( ".ui-button" ).length, 0,
+		"Child widgets are not called when selector is null" );
+
+	element.controlgroup( "option", "items", {
+		"button": "button"
+	});
+	strictEqual( element.children( ".ui-button" ).length, 2,
+		"Correct child widgets are called when selector is updated" );
+});
+
+test( "items: custom selector", function() {
+	expect( 1 );
+	var element = $( ".controlgroup" ).controlgroup({
+		items: {
+			"button": ".button"
+		}
+	});
+	strictEqual( element.children( ".ui-button" ).length, 4,
+		"Correct child widgets are called when custom selector used" );
+});
+
+$.widget( "ui.test", {
+	_create: function (){
+		this.element.addClass( "ui-test ui-button" );
+	},
+	refresh: function() {
+		return;
```

Comment:
widgets used with `controlgroup` must have a refresh method which can be called. i guess i could change this to a `$.noop` instead though.